tox.ini

```diff
@@ -0,0 +1,32 @@
+# Copyright 2022 Canonical Ltd.
```

Comment:
I'm not sure if it is possible, could this be moved into a subdirectory? E.g., using `working-directory`? The reasoning being that this is not really a tox file for the repo but more used as fixtures for the tests. Do you think it should be in a subdir?

Reply:
Having a look weather this is possible! Thank you for the feedback :)